src/features/classification/security-controls/__tests__/SecurityControlsItem.test.js

```diff
@@ -1,21 +1,36 @@
 import React from 'react';
 import SecurityControlsItem from '../SecurityControlsItem';
+import Tooltip from '../../../../components/tooltip';
+import IconInfo from '../../../../icons/general/IconInfo';
 
 describe('features/classification/security-controls/SecurityControlsItem', () => {
     let wrapper;
     let message;
+    let tooltipMessage;
 
     const getWrapper = (props = {}) => shallow(<SecurityControlsItem {...props} />);
 
     beforeEach(() => {
         message = {
-            id: 'msg1',
+            id: 'id1',
             defaultMessage: 'message',
         };
-        wrapper = getWrapper({ message });
+        tooltipMessage = {
+            id: 'id2',
+            defaultMessage: 'message2',
+        };
+        wrapper = getWrapper({ message, tooltipMessage });
     });
 
-    test('should render a SecurityControlsItem with a message', () => {
+    test('should render a SecurityControlsItem with a message and Tooltip', () => {
```

Comment:
nit: The default should probably be the case without the tooltip since that is the most common

Reply:
thanks, will leave this nit as-is.